app/Controllers/userController.php

```diff
@@ -535,10 +535,18 @@ private function retrieveUserDetails($username) {
 		$entryDAO = FreshRSS_Factory::createEntryDao($username);
 		$databaseDAO = FreshRSS_Factory::createDatabaseDAO($username);
 
+		Minz_Configuration::register("user-{$username}",
+		                             join_path(USERS_PATH, $username, 'config.php'),
+		                             join_path(FRESHRSS_PATH, 'config-user.default.php'),
+		                             new FreshRSS_ConfigurationSetter());
+		$userConfiguration = Minz_Configuration::get("user-{$username}");
```

Comment:
You can use the `get_user_configuration($username)` function here

Reply:
Oh I didn't know that function exists!